khmer/utils.py

```diff
@@ -249,24 +248,30 @@ def write_record_pair(read1, read2, fileobj):
     write_record(read2, fileobj)
 
 
+def clean_input_reads(screed_iter):
+    for record in screed_iter:
+        record.cleaned_seq = record.sequence.upper().replace('N', 'A')
+        yield record
+
+
 class ReadBundle(object):
     def __init__(self, *raw_records):
         self.reads = [i for i in raw_records if i]
-        self.cleaned_seqs = [r.sequence.replace('N', 'A') for r in self.reads]
```

Comment:
Agreed!

Yes, the `ReadBundle` class is about aggregation (pairs/singletons of reads).  I'll go update the docs.